scraper.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from bs4 import BeautifulSoup
+import requests
+import sys
+
+
+DOMAIN = 'http://info.kingcounty.gov/'
+PATH = 'health/ehs/foodsafety/inspections/Results.aspx'
+PARAMS = {
+    'Output': 'W',
+    'Business_Name': '',
+    'Business_Address': '',
+    'Longitude': '',
+    'Latitude': '',
+    'City': '',
+    'Zip_Code': '',
+    'Inspection_Type': 'All',
+    'Inspection_Start': '',
+    'Inspection_End': '',
+    'Inspection_Closed_Business': 'A',
+    'Violation_Points': '',
+    'Violation_Red_Points': '',
+    'Violation_Descr': '',
+    'Fuzzy_Search': 'N',
+    'Sort': 'B'
+}
+
+
+def get_inspection_page(**kwargs):
+    url = DOMAIN + PATH
+    params = PARAMS.copy()
+    for k, v in kwargs.items():
+        if k in PARAMS:
+            params[k] = v
+
+    response = requests.get(url, params=params)
+    response.raise_for_status()
+    return response.content, response.encoding
+
+
+def load_inspection_page():
+    with open('inspection_page.html', 'rb') as data:
+        encoded_data = data.read()
+        return encoded_data, 'utf-8'
+
+
+def parse_source(data, encoding='utf-8'):
+    soup_data = BeautifulSoup(data, 'html5lib', from_encoding=encoding)
+    return soup_data
+
+
+def extract_data_listings(soup):
+    import re
+    id_info = re.compile(r'PR0.*~')
+    divs = soup.find_all('div', id=id_info)
+    return divs
```

Comment:
You could save a line by just returning the above line instead of setting a var then returning it.